plugins/dell-dock/fu-dell-dock-usb-usb4.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2021 Intel Corporation.
+ * Copyright (C) 2021 Dell Inc.
+ * All rights reserved.
+ *
+ * This software and associated documentation (if any) is furnished
+ * under a license and may only be used or copied in accordance
+ * with the terms of the license.
+ *
+ * This file is provided under a dual MIT/LGPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ * Dell Chooses the MIT license part of Dual MIT/LGPLv2 license agreement.
+ *
+ * SPDX-License-Identifier: LGPL-2.1+ OR MIT
+ */
+
+#pragma once
+
+#include "config.h"
+
```

Comment:
doesn't this need a fwupdplugin.h include?

Reply:
I think both the `fwupdplugin.h` and `config.h` already included in `common.h`, maybe nothing to be included here? I see nowhere uses `string.h`.

Reply:
For fu-usb-dev that it derived from you should include header first.